components/NavigationBar/src/MDCNavigationBar.m

```diff
@@ -601,6 +599,10 @@ - (void)setTitleView:(UIView *)titleView {
   [self setNeedsLayout];
 }
 
+- (void)setHorizontalTextInsets:(CGFloat)horizontalTextInsets {
+  _horizontalTextInsets = MIN(horizontalTextInsets, CGRectGetWidth(self.bounds) / 2);
```

Comment:
In the case of 200 horizontal inset and both symmetrical then the _width_ of the title view is the diff, so 200 + 200 - 300. So if it was 300 horizontal inset then it would be 300 + 300 - 300.